src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -80,7 +80,7 @@ private set
         /// The task which computes the main rename locations against the original workspace
         /// snapshot.
```

Comment:
It feels this comment needs to be updated to state that it also has the side effect of updating the other map. In some sense this was the "original mistake" in that this function was supposed to be pure and now is not.

Reply:
Documented with a link to https://github.com/dotnet/roslyn/pull/34254#discussion_r267024593. Hopefully this gets cleaned up in the future.